src/sentry/api/bases/organization.py

```diff
@@ -236,6 +236,10 @@ def convert_args(
 
         kwargs["organization_context"] = organization_context
         kwargs["organization"] = organization_context.organization
+
+        # Used for API access logs
+        request._request.organization = organization_context.organization  # type: ignore
```

Comment:
How was this working before 🤔 were we setting `organization` on to the DRF request somewhere else?

Reply:
I don't think many newer parts of the app access `request.organization`, so maybe all the accessors just happened to be region based 👀 All I know is the change was needed for the logs to reflect the correct organizations